scripts/deploy/bump-version.js

```diff
@@ -32,10 +33,19 @@ async function checkoutLatestMaster() {
   await exec('git pull origin master');
 }
 
+/**
+ * @returns {Promise<string[]>}
+ */
+async function getPackagesToBump() {
+  const {stdout} = await exec('git ls-files | grep -e "/package.json$"');
+  const packages = stdout.trim().split('\n').map(filePath => JSON.parse(readFileSync(filePath).toString()));
+  return packages.filter((npmPackage) => npmPackage.version && !npmPackage.private).map(({name}) => name)
```

Comment:
I made it ignore packages that have no version and ignore packages that are private.

Reply:
I'm curious why this error started happening? In the description you said:

```
... failing to graduate every package due to some packages not having a version ...
```
Could another solution be to not use the graduate flag?

Reply:
Using the prerelease flag would add `-alpha` at the end of versions, `conventional-graduate` seems to follow recommended guidelines. I'll specify the packages in lerna.json.

Reply:
That's a good point on the `lerna.json`. Maybe a last idea, could we add `private: true` in the packages that no longer have versions, and then pass [--no-private](https://github.com/lerna/lerna/tree/main/commands/version#--no-private)?

The goal behind this would be to make it easier for devs running lerna commands locally, removing the need to specify scopes to the graduate command.

Reply:
Woops, looks like I didn't notice this option. I'll use that instead.